p03-form-validations/src/Shared/components/Logo/ReactLogo.component.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import "./Reactlogo.styles.css"
+
+const ReactLogo = ({logo, className}) =>{
```

Comment:
Congrats! the idea of the logo component been on the /shared/components is that it could be reusable, so the logo itself is a dependency to render